SmartDeviceLinkTests/RPCSpecs/NotificationSpecs/SDLOnSyncPDataSpec.m

```diff
@@ -38,10 +38,7 @@
                                                      SDLRPCParameterNameOperationName:SDLRPCFunctionNameOnSyncPData}};
 #pragma clang diagnostic pop
 
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
         SDLOnSyncPData* testNotification = [[SDLOnSyncPData alloc] initWithDictionary:dict];
-#pragma clang diagnostic pop
```

Comment:
`SDLOnSyncPData` itself is deprecated, so this creates a warning when running the tests. Please put these back and ensure that you run tests locally to check for any new warnings before checking off that box in the PR description. Since you're changing a bunch of unit tests, you need to run the unit tests, so this PR really requires that box to be checked off too.